schema/models_directory/dft.json

```diff
@@ -2,66 +2,91 @@
     "schemaId": "models-directory-dft",
     "$schema": "http://json-schema.org/draft-04/schema#",
     "title": "density function theory schema",
-    "description": "density function theory schema (deprecated - to be replaced with manifest)",
+    "description": "density function theory schema (using SCI-474)",
     "allOf": [
         {
             "$ref": "../model.json"
         }
     ],
     "properties": {
         "type": {
-            "enum": [
-                "dft"
-            ]
+            "properties": {
+                "name" : {
+                    "type": "string"
+                },
+                "slug": {
+                    "enum": [
+                        "ksdft"
+                    ],
+                    "type": "string"
+                }
+            }
         },
         "subtype": {
-            "enum": [
-                "gga",
-                "lda",
-                "mgga",
-                "ldau"
-            ]
+            "name": {
+                "enum": [
+                    "Local Density Approximation",
+                    "Generalized Gradient Approximation",
+                    "meta-GGA",
+                    "Hybrid Functional",
+                    "Double Hybrid Functional"
+                ]
+            },
+            "slug": {
+                "enum": [
+                    "lda",
+                    "gga",
+                    "mgga",
+                    "hybrid",
+                    "dh"
+                ]
```

Comment:
For the above - we should think about coupling the right enums for name and slug, perhaps